documentation/how-to/outside-layers.en.md

```diff
@@ -75,24 +75,24 @@ The file structure within the **`localized_datasets`** project will reflect the
 ## 3. Localized datasets within QFieldCloud web interface
 
 Once uploaded, these localized datasets will appear in two key places on the QFieldCloud web interface:
-* Within the dedicated **`localized_datasets`** project itself.
+* Within the dedicated **`shared_datasets`** project itself.
```

Comment:
Just noticed sometimes it is bolded and sometimes not. Just to ensure there is some kind of logic there, if there is, please ignore my comment.